pkg/connectors/openapiclient/docs/DefaultApi.md

```diff
@@ -0,0 +1,77 @@
+# \DefaultApi
+
```

Comment:
this folder will not be generated when we use `apiDocs=false` config as mentioned in the earlier comments. Just mentioning it here.

Reply:
Yes.